esmvalcore/preprocessor/_area.py

```diff
@@ -250,9 +250,22 @@ def area_statistics(cube, operator, fx_variables=None):
     grid_areas = tile_grid_areas(cube, fx_variables)
 
     if not fx_variables and cube.coord('latitude').points.ndim == 2:
-        logger.error(
-            'fx_file needed to calculate grid cell area for irregular grids.')
-        raise iris.exceptions.CoordinateMultiDimError(cube.coord('latitude'))
+        if 'grid_latitude' in [coord.standard_name for coord in cube.coords()]:
+            coord_names = ['grid_longitude', 'grid_latitude']
+            cube = guess_bounds(cube, coord_names)
+            # remove lat lon and rename grid_lat grid_lon
+            cube_tmp = cube.copy()
+            for coord_name in coord_names:
+                cube_tmp.remove_coord(coord_name.split('grid_')[1])
+                cube_tmp.coord(coord_name).rename(coord_name.split('grid_')[1])
```

Comment:
it's probably best if you actually do it explicitly ie `cube_tmp.coord('grid_longitude').rename(coord_name.split('longitude')` and the same for latitude